json2cmake/__init__.py

```diff
@@ -35,6 +35,9 @@ def parsecommand(command, directory=os.curdir):
         elif word.startswith('-I'):
             include = word[2:]
             include = os.path.abspath(os.path.relpath(include, directory))
```

Comment:
This line looks like it's just always been broken for relative includes. The [specification](https://clang.llvm.org/docs/JSONCompilationDatabase.html) says that paths can be absolute or relative to the directory, and I think the way to handle that is probably:

```Python
if not os.path.isabs(include):
    include = os.path.join(directory, include)
include = os.path.normpath(include)
```